docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -186,10 +186,11 @@ auth:
 The `teleport-cluster` charts deploys two sets of pods: one for the Auth Service
 and another for the Proxy Service.
 
-`proxy` contains values specific to Proxy Service pods. You can use it to
-set specific values for proxy pods, taking precedence over chart-scoped values.
+`proxy` allows to set chart values only for the proxy Kubernetes resources.
+This is merged with chart-scoped values and takes precedence in case of conflict.
 
-For example, to override the [`postStart`](#postStart) value only for proxy pods:
+For example, to override the [`postStart`](#postStart) value only for proxy pods
+and annotate the proxy Kubernetes Service:
```

Comment:
```suggestion
For example, to override the [`postStart`](#postStart) value only for Teleport Proxy Service pods
and annotate the Kubernetes Service deployed for the Teleport Proxy Service:
```

Suggested for clarity. Teleport engineers will probably know what "proxy pods" means, but users may not.